src/pocketmine/tile/FlowerPot.php

```diff
@@ -66,12 +65,12 @@ public function canAddItem(Item $item) : bool{
 	}
 
 	public function getItem() : Item{
-		return Item::get((int) ($this->namedtag["item"] ?? 0), (int) ($this->namedtag["mData"] ?? 0), 1);
+		return Item::get($this->namedtag->item->getValue(), $this->namedtag->mData->getValue(), 1);
```

Comment:
@jasonwynn10 It's technically possible to unset or redefine almost any variable using reflection classes, I think the plugin should be blamed for their incorrect usages.